packages/settingregistry/src/plugin-schema.json

```diff
@@ -17,6 +17,25 @@
           },
           "type": "array",
           "default": []
+        },
+        "context": {
+          "title": "The application context menu.",
+          "description": "Note: To disable a menu or a menu item,\ncopy it to User Preferences and add the\n\"disabled\" key. The following example will disable ... item:\n{\n  \"contextMenu\": [\n    {\n        \"command\": \"\",\n        \"disabled\": true\n        }\n      ]\n\n}\n\nContext menu description:",
```

Comment:
Is this intentionally here in this template-like form? If yes, maybe use `\"command\": \"...\"`?

Reply:
Thanks for the review - I don't understand this comment. Do you mean I should not set explicitly a command id in to make it template like?

> I set explicitly an example to mimic the shortcut case. I think it is nice to be able to copy-paste an example for the user to try it out.

Reply:
There is no command id in this one. There is in the other one. I actually do not understand why this one is needed, but currently it produces:

```
Note: To disable a menu or a menu item,
copy it to User Preferences and add the
"disabled" key. The following example will disable ... item:
{
  "contextMenu": [
    {
        "command": "",
        "disabled": true
        }
      ]

}

Context menu description:
```

So I thought that maybe using `"command": "...",` instead of `"command": "",` could be better.

Reply:
Indeed the description is not really useful here as that schema is purely for validation with no intention to be displayed. So I drop the example in the description.

Thanks for catching that one.